tests/NamespaceUsagesTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types = 1);
+
+namespace Spaze\PHPStan\Rules\Disallowed;
+
+use PHPStan\File\FileHelper;
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+
+final class NamespaceUsagesTest extends RuleTestCase
+{
+
+	protected function getRule(): Rule
+	{
+		return new NamespaceUsages(
+			new DisallowedNamespaceHelper(new FileHelper(__DIR__)),
+			[
+				[
+					'namespace' => 'Inheritance\Sub',
```

Comment:
Add more tests, also for disallowed "parent" namespace, `Inheritance` in this case. Feel free to create new classes, if needed.

Reply:
Done